apps/whale-api/src/module.api/stats.controller.ts

```diff
@@ -166,17 +171,20 @@ export class StatsController {
     const utxo = burnInfo.amount
     const account = findTokenBalance(burnInfo.tokens, 'DFI')
     const address = utxo.plus(account)
+    const payback = burnInfo.paybackburn.length !== 0
+      ? burnInfo.paybackburn.map(i => new BigNumber(i)).reduce((a, b) => a.plus(b))
```

Comment:
Given it's ["0.00000456@DFI"], we need to split, and filter @DFI values only?

Reply:
Let me work on these

Reply:
Committed the change as https://github.com/JellyfishSDK/jellyfish/pull/1580/commits/8c84738565fe6f3161ee362cf90411fa58cc80d0

Reply:
Open issue describing the problem: https://github.com/JellyfishSDK/jellyfish/issues/1600